lib/src/infrasructure/repositry/home_repositry/get_banner_repositry.dart

```diff
@@ -46,7 +45,7 @@ class GetBannerRepositry implements IGetBannerRepositry {
             "Unexpected response type: ${response.data.runtimeType}");
```

Comment:
AppConstants.bannerImageBaseUrl = responseData['image_base_url'];

Put this line to outside of the loop